crates/env/src/lib.rs

```diff
@@ -102,16 +102,18 @@ mod tests;
 #[doc(inline)]
 pub use self::engine::off_chain::test_api as test;
 
+pub use pallet_contracts_uapi::{
```

Comment:
How will the docs look, should we do `#[doc(inline)]` for these types?

Reply:
ah did not know we could do that, let me  test to see how the gen docs looks like